internal/discord/discord.go

```diff
@@ -0,0 +1,36 @@
+package discord
```

Comment:
Nitpick can we throw this into alert/discord instead as it's related logic to alerting :)